users_ldap_push/models/res_company_ldap.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module copyright (C) 2015 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import _, models, fields, api, exceptions
+
+
+class ResCompanyLdap(models.Model):
+    _inherit = 'res.company.ldap'
+
+    create_ldap_entry = fields.Boolean('Create ldap entry', default=True)
+    create_ldap_entry_base = fields.Char(
+        'Create ldap entry in subtree',
+        help='Leave empty to use your LDAP base')
+    create_ldap_entry_objectclass = fields.Char(
+        'Object class', default='account',
+        help='Separate object classes by comma if you need more than one')
+    create_ldap_entry_field_mappings = fields.One2many(
+        'res.company.ldap.field_mapping', 'ldap_id', string='Field mappings')
+
+    @api.model
+    def default_get(self, fields_list):
+        result = super(ResCompanyLdap, self).default_get(fields_list)
+        if 'create_ldap_entry_field_mappings' in fields_list:
+            result.setdefault(
+                'create_ldap_entry_field_mappings',
+                [
+                    (0, 0, {
+                        'field_id':
+                        self.env.ref('base.field_res_users_login').id,
+                        'attribute': 'userid',
+                        'use_for_dn': True,
+                    }),
+                ])
```

Comment:
is there a reason why you're not defining a dedicated default method and passing it as the `default` kwarg to the field?